lib/models/questionnaire_store.dart

```diff
@@ -31,6 +32,17 @@ class QuestionnaireStore {
     _questionnaires.remove(questionnaire);
   }
 
+  void clear() {
+    _questionnaires.clear();
+  }
+
+  void updateAll(QuestionCatalog questionCatalog) {
+    _questionnaires.forEach((questionnaire, value) {
+      questionnaire.questionCatalogF = questionCatalog;
+      questionnaire.updateLanguage();
+    });
+  }
```

Comment:
This method doesn't make it clear that it should only be called when the language changed.
Perhaps remove it and introduce:

`Iterable<Questionnaire> get items => _questionnaires.keys;`

So you can then loop over it outside of this class yourself.